packages/arb-token-bridge-ui/src/components/TransactionHistory/TransactionsTableExternalLink.tsx

```diff
@@ -0,0 +1,41 @@
+import { Fragment, PropsWithChildren, useState } from 'react'
+
+import { ExternalLink } from '../common/ExternalLink'
+import { Transition } from '@headlessui/react'
+import { ArrowTopRightOnSquareIcon } from '@heroicons/react/24/outline'
+
+export const TransactionsTableExternalLink = ({
```

Comment:
on hover we show a link to the account on that network

<img width="348" alt="image" src="https://github.com/OffchainLabs/arbitrum-token-bridge/assets/47932951/55b950d6-20b6-4e3e-969e-535d9fa4540f">